scrapers/covid-19/historical.js

```diff
@@ -147,6 +201,20 @@ const getHistoricalCountryDataV2 = (data, query, province = null, lastdays = 30)
 			});
 		});
 	});
+	/*
```

Comment:
Something else to consider: We don't want to return the fake povince with the actual data, especially when it is used. In this list of provinces returned
<img width="332" alt="image" src="https://user-images.githubusercontent.com/4297028/99879037-a8823280-2bd7-11eb-858b-45e91fec69b1.png">
it shouldn't be there

Reply:
I have taken care of that with the line of code below
```js
if (standardizedCountryName.toLowerCase() === 'canada' && province === null) {
		for (let i = 0; i < provinces.length; i++) {
			if (provinces[i] === 'recovered-aggregate') {
				provinces.splice(i, 1);
				break;
			}
		}
	}
```